<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoListDialogClass</class>
 <widget class="QDialog" name="AutoListDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>844</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>自动添加</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Money.qrc">
    <normaloff>:/Images/add_user.png</normaloff>:/Images/add_user.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="4">
    <widget class="MyTableView" name="tableView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="buttonAdd">
     <property name="icon">
      <iconset resource="Money.qrc">
       <normaloff>:/Images/edit_add.png</normaloff>:/Images/edit_add.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="buttonDel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="icon">
      <iconset resource="Money.qrc">
       <normaloff>:/Images/edit_remove.png</normaloff>:/Images/edit_remove.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="buttonClose">
     <property name="icon">
      <iconset resource="Money.qrc">
       <normaloff>:/Images/exit.png</normaloff>:/Images/exit.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyTableView</class>
   <extends>QTableView</extends>
   <header>MyTableView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Money.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonClose</sender>
   <signal>clicked()</signal>
   <receiver>AutoListDialogClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>805</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>712</x>
     <y>-12</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
